samples/watson-train-serve/README.md

```diff
@@ -0,0 +1,56 @@
+# Watson Train and Server.
+
+This pipeline runs training, storing and deploying a Tensorflow model with MNIST handwriting recognition using [IBM Watson Machine Learning](https://www.ibm.com/cloud/machine-learning) service.
```

Comment:
maybe add an extra sentence to point to the original example, so the users know what's going on with this example.

Reply:
sure